From 9daf706680e02c2964f93e6c1941693f92a619e2 Mon Sep 17 00:00:00 2001
From: Kihwal Lee <kihwal@apache.org>
Date: Wed, 26 Oct 2016 09:18:45 -0500
Subject: [PATCH 2076/2848] HDFS-11053. Unnecessary superuser check in
 versionRequest(). Contributed by Kihwal Lee.
 Updated CHANGES.txt (cherry picked from commit
 e90af4a89ba2741aeca20ead47699587b14f0de8)

(cherry picked from commit fdd645be6b5bb3a19b3c3530d03907d563abe257)

 Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

Change-Id: Ic0548dcc977dbf8a9b9f7eda21b5502d5aadebe6
---
 .../hdfs/server/namenode/NameNodeRpcServer.java    |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java
index 79ae65b..d158727 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java
@@ -1288,7 +1288,6 @@ public void errorReport(DatanodeRegistration nodeReg,
   @Override // DatanodeProtocol, NamenodeProtocol
   public NamespaceInfo versionRequest() throws IOException {
     checkNNStartup();
-    namesystem.checkSuperuserPrivilege();
     return namesystem.getNamespaceInfo();
   }
 
-- 
1.7.9.5

